autokeras/hypermodel/processor.py

```diff
@@ -107,7 +167,91 @@ def transform(self, hp, data):
         # Returns
             A DataLoader instance.
         """
-        # channel-wise normalize the image
-        def normalize(x):
-            return (x - self.mean) / self.std
-        return data.map(normalize)
+        x = nest.flatten(x)[0]
+        return (x - self.mean) / self.std
+
+    def output_types(self):
+        return tf.float64,
+
+    def output_shape(self):
+        return self.mean.shape
+
+
+class TextToSequenceVector(HyperPreprocessor):
```

Comment:
It isn't very clear what a "sequence vector" would be. Perhaps `TextToIntSequence` would be clearer.